File: NetworkMonitorApi/ClientApp/src/app/models/post.ts
import { Comment } from "./comment";

//export class Post {
//  postId: number;
//  title: string;
//  content: string;
//  blogId: number;
//  likes: number;
//  comments: Comment[];
//}

export class Post {
  public postId: number;
  public title: string;
  public content: string;
  public blogId: number;
  public userId: string;
  public author: string;
  public dateCreated: string;
  public dateModified: string;
  public likes: number;
  public disLikes: number;
  public comments: Comment[];
}

File: NetworkMonitorApi/Models/Post.cs

using System;
using System.Collections.Generic;

namespace NetworkMonitorApi.Models
{
    public class Post
    {
        public int PostId { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public int BlogId { get; set; }
        public string UserId { get; set; }
        public string Author { get; set; }
        public DateTime DateCreated{get;set;}
        public DateTime DateModified { get; set; }
        public int Likes { get; set; }
        public int DisLikes { get; set; }
        public ICollection<Comment> Comments { get; set; }
    }
}
